packages/backend-modules/republik-crowdfundings/graphql/schema-types.js

```diff
@@ -94,6 +94,8 @@ type PackageOption {
   minUserPrice: Int!
   userPrice: Boolean!
   autoPay: Boolean
+  fixedPrice: Boolean!
+  payMoreSuggestion: Boolean!
```

Comment:
I don't see any resolver changes: will these fields automagically show up?

Reply:
They do (I tested it), see https://github.com/republik/plattform/pull/516/files/5cd91fc92ab2dcae30603dc0816c389a07237cf5#r1263865068